CMakeLists.txt

```diff
@@ -84,6 +84,7 @@ tvm_option(USE_BLAS "The blas library to be linked" none)
 tvm_option(USE_MKL "MKL root path when use MKL blas" OFF)
 tvm_option(USE_MKLDNN "Build with MKLDNN" OFF)
 tvm_option(USE_DNNL_CODEGEN "Enable MKLDNN (DNNL) codegen" OFF)
+tvm_option(USE_CCACHE "Use the 'ccache' compiler wrapper when building TVM" AUTO)
```

Comment:
maybe we want this off by default since it would take disk? or alternatively, can you change tests/scripts/task_build.py to flip this off in CI build (we have sccache there)?

Reply:
For non-CI environments I think either choice is reasonable.  "auto" was supposed to be the default all along, but AFAICT a bug meant it was basically acting like "OFF".  Thoughts?

For CI environments, I'm happy to make whatever changes you / @mehrdadh would like.  Just let me know.

Reply:
I think by default should be OFF since we want to build from scratch in CI. Actually it may doesn't matter since it's in docker.

Reply:
Thanks.  Sounds like the best plan is:
- Let `USE_CCACHE` continue to default to `AUTO`.
- I'll modify `tests/scripts/task_build.py` to set `USE_CCACHE=OFF`, so that CI doesn't pointlessly waste storage space.

If/when we want to enable `ccache` or` sccache` in CI, we can just do another PR.

Reply:
This is moot now they the PR removes `USE_CCACHE` entirely.